pkg/url/url.go

```diff
@@ -92,6 +93,66 @@ func List(client *occlient.Client, componentName string, applicationName string)
 	return urlList, nil
 }
 
+// List returns all URLs for given component. The results can further be narrowed
+// down if a component name is provided, which will only list URLs for the
+// given component
+func List(client *occlient.Client, localConfig *config.LocalConfigInfo, componentName string, applicationName string) (UrlList, error) {
```

Comment:
A few comments in the function would be helpful. For example before the two `for` loops that work on `routes` and `localConfigUrls`, what `found` bool is meant for.

Reply:
the comment inside `if !found` block is trying to explain it